frontend/scripts/index.ejs

```diff
@@ -9,4 +9,6 @@
 <div id="app-root-container"></div>
 </body>
 <script type="text/javascript" src="//cdn.transifex.com/live.js"></script>
+<!-- Rangetouch to fix <input type="range"> on touch devices (see https://rangetouch.com) -->
+<script src="https://cdn.rangetouch.com/1.0.5/rangetouch.js"></script>
```

Comment:
cant we use https://github.com/sampotts/rangetouch#node--npm instead?

Reply:
Done.